extra-android/android-file-transfer/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=android-file-transfer
+PKGSEC=utils
+PKGDES="Android MTP client with minimalistic UI"
+PKGDEP="qt-5 fuse libxkbcommon hicolor-icon-theme file"
+BUILDDEP="cmake"
```

Comment:
We ship CMake in `devel-base`, no need to depend on it directly.

```suggestion

```

Reply:
I've removed it in 05856fe, thanks.